pocs/observatory.py

```diff
@@ -490,6 +526,10 @@ def get_standard_headers(self, observation=None):
         Returns:
             dict: The standard headers
         """
+
+        if not self.can_observe:
```

Comment:
This check isn't necessary as should be handled by the `assert` on line 536 below.